### PR TITLE
Translate localisation strings to Polish

### PR DESCRIPTION
**Pull request by blazeblazeblaze**
This PR translates localisation strings to Polish as per request. 
Issue: https://github.com/aahung/Unshaky/issues/111

@youshy might want to have a look before it's merged.

### CODE REVIEW

File: Unshaky/pl.lproj/Counter.strings [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
 /* Class = "NSTableColumn"; headerCell.title = "Key"; ObjectID = "Y81-5k-LdE"; */
-"Y81-5k-LdE.headerCell.title" = "Key";
+"Y81-5k-LdE.headerCell.title" = "Klawisz";
 
 /* Class = "NSTextFieldCell"; title = "-"; ObjectID = "Yte-dc-JdH"; */
 "Yte-dc-JdH.title" = "-";
 
 /* Class = "NSTableColumn"; headerCell.title = "Dismissed"; ObjectID = "m7I-RF-TDD"; */
-"m7I-RF-TDD.headerCell.title" = "Dismissed";
+"m7I-RF-TDD.headerCell.title" = "Uniknięto";
```
Comment (youshy) [inline Unshaky/pl.lproj/Counter.strings]: Maybe `Odkliknięto` or `Odrzucono` might be a better one?

File: Unshaky/pl.lproj/Preference.strings [hunk 2]
```diff
@@ -1,66 +1,66 @@
...
 /* Class = "NSTextFieldCell"; title = "-"; ObjectID = "Vl3-75-cL7"; */
 "Vl3-75-cL7.title" = "-";
 
 /* Class = "NSMenuItem"; title = "Item 1"; ObjectID = "WY9-It-Jdo"; */
-"WY9-It-Jdo.title" = "Item 1";
+"WY9-It-Jdo.title" = "Element 1";
 
 /* Class = "NSButtonCell"; title = "Aggressive mode (experimental)"; ObjectID = "Zqq-UB-oIK"; */
-"Zqq-UB-oIK.title" = "Aggressive mode (experimental)";
+"Zqq-UB-oIK.title" = "Try agresywny (eksperymentalnie)";
```
Comment (youshy) [inline Unshaky/pl.lproj/Preference.strings]: `Tryb`

File: Unshaky/pl.lproj/Counter.strings (hunk 1 above)
Comment (blazeblazeblaze) [reply Unshaky/pl.lproj/Counter.strings]: Makes sense, will do

File: Unshaky/pl.lproj/Preference.strings (hunk 2 above)
Comment (blazeblazeblaze) [reply Unshaky/pl.lproj/Preference.strings]: Good catch

**Comment by aahung** [comment]
Thanks guys!

**Pull request merged**